scripts/build.js

```diff
@@ -276,23 +289,35 @@ const inlineRegexSourcePlugin: Plugin = {
  * is a waste of CPU and memory, and it causes the JS thread to be block for roughly 200ms during page load.
  *
  * @see https://github.com/PrismJS/prism/issues/2768
+ *
+ * @type {Plugin}
  */
-const lazyGrammarPlugin: Plugin = {
+const lazyGrammarPlugin = {
```

Comment:
We still need the build step, not only because of types (we could do it in an NPM script), but because of this plugin (see the comment above).

Reply:
Lazy grammars are evaluated when needed, so not sure why we need a build step for them?

Reply:
Oh, I might mislead you with the comment that we need the build step only because of that plugin. My bad. We have other plugins to apply, as well as build types, calculating file sizes (to be used on the website's Download page), building a treeview icon font, etc. So, wrapping bare grammar objects with functions is just one step of many other build steps. Probably, the build step should also be revised, but I'd leave it for another PR.

For context, this is why that plugin was introduced: https://github.com/PrismJS/prism/issues/2768